packages/commonwealth/client/styles/pages/notifications/notification_row.scss

```diff
@@ -81,11 +80,16 @@
 
   .comment-body-mark-as-read {
     display: inline;
-    color: $neutral-500;
+    color: $primary-300;
     margin-left: 8px;
   }
 
   .comment-body-mark-as-read:hover {
     color: $primary-600;
   }
 }
+
+.NewNotificationRow {
+  background: $neutral-50;
+  border-bottom: 1px solid $neutral-300;
+}
```

Comment:
we have class `unread` above in `NotificationRow`  - isn't it the thing that should be handled in this PR? Is it necessary to create another class? If it is, follow the pattern, so someting like `&.new`

Reply:
Addressed